packages/vitest/package.json

```diff
@@ -61,7 +61,7 @@
     "local-pkg": "^0.4.1",
     "tinypool": "^0.1.1",
     "tinyspy": "^0.2.10",
-    "vite": ">=2.7.13"
+    "vite": ">=2.8.0"
```

Comment:
What is the reason we were using `>=` instead of `^` here?

Reply:
Yeah good point. They are basically the same atm. We should not bump peerDeps until we are relying on 2.8's feature later. Since we'd better keep allowing users to still use Vitest along with Vite 2.7 (otherwise there will be two vite and cause the misalignment)

Reply:
Where is vite as a peerDependency in vitest? I don't see where I shouldn't be bumping to 2.8 in this PR

Reply:
Oh, I mean deps. Basically, if users have a fixed 2.7 in their package.json and we bump it to 2.8, it would cause two versions of Vite, which I think it's not ideal. Bumping our devDeps in our root package would be enough

Reply:
Gotcha 👍🏼